interop_tests/dockerfiles/Dockerfile

```diff
@@ -0,0 +1,16 @@
+FROM python:3.10.7
+
+# Clone interop-tests repo
```

Comment:
Now that this is a python package, we can install it via `pip`:

```
RUN pip3 install git+https://github.com/rhpit/interop-tests.git@main
```

Then when installed, it will install `openshift-python-wrapper` as it will be defined in the `setup.cfg`.